index.js

```diff
@@ -499,20 +499,16 @@ function signAsync (opts) {
     })
     .then(function () {
       // Pre-sign operations
-      var promise = Promise.resolve()
-      if (opts.version ? compareVersion(opts.version, '1.1.1') >= 0 : true) {
+      if (opts.entitlements != null && (opts.version == null || compareVersion(opts.version, '1.1.1') >= 0)) {
```

Comment:
Sorry, up to standard style it is better, yes, should be no `!= null`. Although I don't like it due to obvious reasons.